Monika After Story/game/script-anniversary.rpy

```diff
@@ -674,6 +675,34 @@ label anni_7:
     m 1ekbsa "Here's to another year together, and all the ones to follow."
     return "unlock"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel='anni_8',
+            prompt="8 Years",
+            category=["anniversary"],
+            action=EV_ACT_QUEUE,
+            start_date=store.mas_anni.build_anni(years=8),
+            end_date=store.mas_anni.build_anni_end(years=8)
+        ),
+        skipCalendar=False
+    )
+
+label anni_8:
+    m 1hua "I'm sure you know what today is, [player]..."
+    m 3hublb "That's right! Happy anniversary, [mas_get_player_nickname()]!"
+    m 3ekblb "You know...when a relationship has lasted this long, sometimes couples start to worry that things might grow stale."
+    m 1lkbla "Eight years is a long time to learn someone's habits, and to build your lives around each other."
+    m 1lkc "Maybe...{w=0.5}you've worried about similar things, [player]?"
+    m 2ekb "I mean--{w=0.2}I don't want to assume!{w=0.3} But I was thinking about it earlier, and I wanted to tell you something."
```

Comment:
I think it could be cute if we added some blush to `But I was thinking about it earlier, and I wanted to tell you something`, maybe with an extend?
```suggestion
    m 2ekb "I mean--{w=0.2}I don't want to assume!{w=0.3} {nw}"
    extend 2ekbla "But I was thinking about it earlier, and I wanted to tell you something."
```